content/academy/anti_scraping/proxies/apify_proxy.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Apify Proxy
+description: Learn about Apify Proxy, a powerful service which handles all of the difficult parts of scraping with proxies.
+menuWeight: 2
+category: courses
+paths:
+- anti-scraping/proxies/apify-proxy
+---
+
+# [](#apify-proxy) Apify Proxy
+
+<!-- using Apify Proxy in the SDK -->
```

Comment:
This should probably be part of the Apify platform course, not the web scraping courses.